### PR TITLE
change(web): revert #11174, which loads keyboards before initializing the OSK 🍒 🏠

### PR DESCRIPTION
**Pull request by mcdurdin**
Debug-mode keyboards are not safe to load before the OSK is initialized due to certain debug APIs being published _through_ `keyman.osk`.  The cleanest way to remedy #11962 is thus to revert the PR that attempts to load keyboards before the OSK is initialized.

Fixes: #11962
Reverts: #11174
Reverts: #11258
Cherry-pick-of: #12015

## User Testing

**TEST_REPRO**:  Attempt to reproduce #11962 using Keyman for Android.

- Install the package provided within that issue.
- Force-quit the app.
- Relaunch the app.
    - If the issue is not properly fixed, an error notification will appear on app-relaunch.

**TEST_ANDROID**:   Do simple general-use testing with Keyman for Android and report back if any odd behaviors occur.

**TEST_IOS**: Do simple general-use testing with Keyman for iOS and report back if any odd behaviors occur.

**TEST_WEB**:  Do simple general-use testing with the "Test unminified Web" test page and report back if any odd behaviors occur.

**TEST_DEVELOPER**: Verify that the Keyman Developer Web Debugger is working correctly.

### CODE REVIEW

**Review by darcywong00 — APPROVED** [review]
lgtm

File: web/src/engine/osk/src/views/oskView.ts [hunk 1]
```diff
@@ -252,22 +251,10 @@ export default abstract class OSKView
...
-    if(modelState) {
-      this.bannerController.selectBanner(modelState);
-    }
-
     this.setBaseMouseEventListeners();
     if(this.hostDevice.touchable) {
       this.setBaseTouchEventListeners();
     }
-
-    this._Box.style.display = 'none';
```
Comment (jahorton) [inline web/src/engine/osk/src/views/oskView.ts:269]: Given the context of the comment on #12015, I believe that this specific line should not be reverted.

Comment (jahorton) [reply web/src/engine/osk/src/views/oskView.ts:269]: Given further discussion there, it sounds like we'll go ahead and revert it for now, with a focused... "unreversion"?... of this as a quick followup.

**Review by jahorton — APPROVED** [review]

**Comment by dinakaranr** [comment]
# Test Results

I tested this issue with the attached "Keyman 17.0.328" build on the Android 14 & iPhone, web(Chrome & Firefox), and keyboard developer. Here is my observation.

* **TEST_REPRO (Passed):** 
1. Install the "Keyman 17.0.328-test-12040" build.
2. Relaunch the app after force quit. 
3. No error notification appears when forced to quit the app.

* **TEST_ANDROID (passed):**
1. Installed the "Keyman-17.0.328.apk" file and gave all permissions to the application.
2. Checked the "Enable Keyman as system-wide keyboard" and set the keyboard as the default keyboard box on the settings page.
3. Open the Keyman app. Enable the "Predictions" and install the "Dictionary."
4. Added a sentence to the notepad and Chrome (search).
5. Enter a word and observe that the words are suggested and highlighted on the banner. 
6. The keyboard works on portrait and landscape viewport. Keyboard set as system keyboard.
7. Installed/uninstalled multiple keyboards.

It looks like it was expected.

* **TEST_IOS (passed):**
1. Installed the "Keyman-17.0.328" build from the testflight app and gave all permissions to the application.
2. Checked the "Enable Keyman as system-wide keyboard" and set the keyboard as the default keyboard box on the settings page.
3. Open the Keyman app. Enable the "Predictions" and install the "Dictionary."
4. Added a sentence to the notepad and Chrome (search).
5. Enter a word. Observe that the words are suggested and highlighted on the banner. 
6. The keyboard works on portrait and landscape viewports. Keyboard set as system keyboard.
7. Installed/uninstalled multiple keyboards.
It looks like it was expected.

* **TEST_WEB (Passed):**
1. Navigate to the "Tests unminified Keymanweb" --> "KeymanWeb Sample Page - Unminified Source" pages.
2. "English - English": It works well in the text box and OSK keyboard.
3. "Lao - Lao Basic": It works well in the text box and OSK keyboard.
4. "English - Classic 10-key": OSK board appears but it is blank now. An alert window appears for very few seconds.
It works in Khmer and French keyboards too. Thank you. 

* **TEST_DEVELOPER (Passed):**
1. Added the Amharic & Cameroon keyboards loaded the .kmn, and pressed F7 to compile.
2. SUITE_DEBUGGER: Test new core-based debugger - Compile and recompile the keyboard
3. SUITE_SERVER: Test new Keyman Developer Server - Keyboard launched on the browser with OSK.
I tested the above two test suites and work as per expected. Thanks

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.329](https://keyman.com/downloads/releases/stable/17.0.329)

**Pull request merged**